src/Reader.php

```diff
@@ -34,7 +41,7 @@
      *
      * @return array An array of Annotations.
      */
-    function getClassAnnotations(\ReflectionClass $class);
+    public function getClassAnnotations(ReflectionClass $class) : array;
```

Comment:
I agree with adding `public ` access modifier. But would it not be better to split this into a separate PR, which gets applied before this one, to every branch?